src/layer/domain/router/model/eav/value/fetch.ts

```diff
@@ -24,6 +24,6 @@ export class FetchResponse {
     name => this.xhr.getResponseHeader(name);
   public readonly document: Document =
     this.xhr.responseType === 'document'
-      ? this.xhr.responseXML!.cloneNode(true) as Document
+      ? this.xhr.responseXML as Document
```

Comment:
Since XHR objects can be cached, they must be cloned. Why cloneNode fails?

Reply:
I'm not sure. As `2.` shows in console,

```
[Exception... "Unexpected error"  nsresult: "0x8000ffff (NS_ERROR_UNEXPECTED)"  location: "JS frame :: debugger eval code :: <TOP_LEVEL> :: line 1"  data: no]
```

It seems that something went wrong when accessing memory out side add-on context.